common-bsp/recipes-kernel/device-tree-overlays/device-tree-overlays_git.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "Device Tree Overlays for bb.org boards"
+
+LICENSE = "MIT"
```

Comment:
Yes, try:

LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/GPL-2.0;md5=801f80980d171dd6425610833a22dbe6"

That will use the gpl file in OE-core